### PR TITLE
fix(ObjectViewer): Fixes object viewer style

### PR DESCRIPTION
**Pull request by talend-glorieux**
**What is the problem this PR is trying to solve?**
Object viewer has styles issues in grid mode since changes on the tree view mode were not isolated. 

**What is the chosen solution to this problem?**
* Isolate each view style. 
* Isolate long text case into it's own story. 

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/components/src/ObjectViewer/Table/Table.component.js [hunk 1]
```diff
@@ -87,7 +87,12 @@ function Table({ flat, data, ...props }) {
 								const path = getAbsolutePath(index, key, flat);
 								return (
 									<td key={j}>
-										<JSONLike data={flattenRow[key]} {...props} jsonpath={path} />
+										<JSONLike
+											data={flattenRow[key]}
+											{...props}
+											jsonpath={path}
+											nativeValueClassName={'native-value-table'}
```
Comment (jmfrancois) [inline packages/components/src/ObjectViewer/Table/Table.component.js]: `nativeValueClassName="native-value-table"`

File: packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ function stopAndSelectWithEnterOrSpace(event, { onSelect, jsonpath }) {
 	}
 }
 
-export function NativeValue({ data, edit, onSelect, onChange, jsonpath }) {
+export function NativeValue({ data, edit, className, onSelect, onChange, jsonpath }) {
```
Comment (jmfrancois) [inline packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js]: rename as `themeAttr`

**Comment by jmfrancois** [comment]
The demo on containers seems broken.
Could you take a look ?

File: packages/components/src/ObjectViewer/JSONLike/JSONLike.scss [hunk 3]
```diff
@@ -91,6 +91,13 @@
 		color: $dove-gray;
 	}
 
+	.native-value-table {
```
Comment (jsomsanith) [inline packages/components/src/ObjectViewer/JSONLike/JSONLike.scss]: This is used in Table component, but it means that this component has to know the implementation of JSONLike component.
IMO 
* this should be defined in Table.scss
* JSONLike takes a native value className and doesn't resolve it from the theme, but set it directly
* Table component resolves from its theme and pass it to JSONLike

**Pull request merged**